book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -245,7 +245,7 @@ please contact us at email.support@github.com
 </div>
 ----
 
-這個解決方案分別採納了兩個分支中的各一部分內容，並且完整地移除了 `<<<<<<<`、`=======` 和 `>>>>>>>` 這些標記行。
+這個解決方案分別採納了兩個分支中的各一部分內容，並且完整地移除了 `>>>>>` 這些標記行。
```

Comment:
mis-deleting few words about conflict marks?

Reply:
Good catch. My regexp for conversion was too greedy. Will revert.